deployments/multiple-ragapps/docker-compose.yml

```diff
@@ -5,18 +5,66 @@ services:
       - 80:80
     labels:
       - traefik.enable=true
+      # API and dashboard
       - traefik.docker.network=ragapp-network
-      # Default username: admin, password: admin
-      - traefik.http.middlewares.admin-auth.basicauth.users=${USERNAME:-admin}:${HASHED_PASSWORD:-$$apr1$$3gG256rA$$BomHrC/uMhL807k/t7Wd6/}
-    volumes:
+      # Keycloak OpenID plugin configuration
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.KeycloakURL=http://localhost/auth
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.ClientID=ragapp
+      # Secret from the example ragapp keycloak realm
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.ClientSecret=jh4yDlJ5QsBSafVNaqqLxEG3vKcYyNgD 
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.KeycloakRealm=ragapp
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.Scope=openid
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.TokenCookieName=AUTH_TOKEN
+      - traefik.http.middlewares.my-keycloakopenid.plugin.keycloakopenid.UseAuthHeader=false
+    volumes:  
       - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ./traefik/traefik.yml:/traefik_config.yml:ro
     command:
-      - --providers.docker
-      - --providers.docker.exposedbydefault=false
-      - --entrypoints.http.address=:80
-      - --accesslog
-      - --log
-      - --api
+      - --configFile=/traefik_config.yml
+    networks:
+      - ragapp-network
+
+  # IMPORTANT: This keycloak instance is for development purposes only
+  # For production, you should use a more secure setup: https://www.keycloak.org/server/configuration-production
```

Comment:
better move this into the README and generally update the README with the most common use cases:

1. Login as admin
2. Login as user (there should be an example user already configured)

KISS - just one line - also add an link to the doc to create/modify users